src/App.js

```diff
@@ -9,6 +9,10 @@ const App = () => {
 
   const gamePlay = (index) => {
       let updatedSquares = [...squares]
+
+      if (squares[index]){
+        return
```

Comment:
Careful with leaving empty returns. This can create buggy results in the future.  Consider refactoring so that you are returning something

Reply:
Updated changes